### ISSUE TITLE
Failed test due to javascript error not reported as failed

### ISSUE THREAD

**Issue by guillegr123** [postmanlabs/newman#371]
Suppose I get this response:

`{y:10}`

I have a test similar to this (being explicit and shorter):

```
var response = JSON.parse(responseBody); 
tests['Response contains y value'] = response.y;
tests['Response contains w value inside x'] = response.x.w;
```
(I know that I should test before if x is whitin the response, but sometimes this kind of situations slip away)

The item in postman collection that has this test fails, and newman returns 1. This is also displayed when newman is run from the command line. However, in the logs that newman outputs, there is no easy way to identify that an item in the collection failed, because it says that there were 1 test, and there are 0 failures.

```
"totalPassFailCounts": {
                "pass": 1,
                "fail": 0
            }
```

**Comment by abhijitkane** [comment]
@guillegr123 `{y:10}` is invalid JSON. It needs to be `{"y":10}`
Screenshot attached.
<img width="537" alt="screen shot 2016-03-31 at 21 53 00" src="https://cloud.githubusercontent.com/assets/681190/14182865/faffdda4-f78a-11e5-9c45-5a8b1af8df8f.png">

**Comment by guillegr123** [comment]
Indeed, sorry, it is {"y":0}. I forgot to put the quotes when was describing the issue.

**Comment by shamasis** [comment]
The upcoming newman v3 does this a bit differently and includes all forms of errors. Closing this since now script errors are included in failures for newman v3.0 beta. https://github.com/postmanlabs/newman/tree/feature/v3

Please reopen if otherwise.
